### PR TITLE
[hotfix] Fix typo in elastic_scaling.md

### PR DESCRIPTION
**Pull request by noahshpak**
Minor documentation fix: `ALL_EXCHANGES_BLOCKING`  != `ALL-EXCHANGES-BLOCKING`

### CODE REVIEW

File: docs/content/docs/deployment/elastic_scaling.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ To automatically decide parallelisms for operators with Adaptive Batch Scheduler
 #### Configure to use Adaptive Batch Scheduler
 To use Adaptive Batch Scheduler, you need to:
 - Set `jobmanager.scheduler: AdaptiveBatch`.
-- Leave the [`execution.batch-shuffle-mode`]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) unset or explicitly set it to `ALL-EXCHANGES-BLOCKING` (default value) due to ["ALL-EXCHANGES-BLOCKING jobs only"](#limitations-2).
+- Leave the [`execution.batch-shuffle-mode`]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) unset or explicitly set it to `ALL-EXCHANGES-BLOCKING` (default value) due to ["ALL_EXCHANGES_BLOCKING jobs only"](#limitations-2).
```
Comment (reswqa) [inline docs/content/docs/deployment/elastic_scaling.md:169]: We should also change the `docs/content.zh/docs/deployment/elastic_scaling.md`.

**Review by reswqa — CHANGES_REQUESTED** [review]
Thanks for the contribution, I left some comments, please take a look.

**Comment by reswqa** [comment]
BTW, the name of this commit can be changed to `[hotfix] Fix typo in elastic_scaling.md`.

**Comment by reswqa** [comment]
Also cc @wanglijie95 for the double review.

**Comment by wanglijie95** [comment]
Thanks for creating this PR @noahshpak ! @reswqa is right, we should also fix the Chinese document: `docs/content.zh/docs/deployment/elastic_scaling.md`

**Comment by reswqa** [comment]
What's the progress of this pull request? FLINK-30860 will also modify `elastic_scaling.md`, if you do not have time to fix this, I will do this work in that ticket to avoid conflict.

**Comment by noahshpak** [comment]
Sorry for the delay here--other doc should be fixed now too! Thanks for taking a look

**Review by wanglijie95 — APPROVED** [review]
LGTM.

**Comment by wanglijie95** [comment]
@flinkbot run azure

**Comment by wanglijie95** [comment]
@flinkbot run azure

**Comment by wanglijie95** [comment]
Thanks @noahshpak @reswqa, merged.

**Pull request merged**